src/css/objects/_media.scss

```diff
@@ -33,42 +34,43 @@
 
     &::before {
       box-sizing: border-box;
-      content: '';
+      content: "";
       display: block;
-      padding-top: 66.6666667%; // 3:2 by default
       position: relative;
     }
-
-    &--cinematic::before {
-      padding-top: 42.5531915%;
-    }
-
-    &--four-by-three::before {
-      padding-top: 75%;
-    }
-
-    &--widescreen::before {
-      padding-top: 56.25%;
-    }
-
-  }
-
-  &__img {
-    box-sizing: border-box;
-    left: 50%;
-    position: absolute;
-    top: 50%;
-    transform: translate(-50%, -50%);
   }
 
   &__content {
+    @include clearfix;
+
     display: block;
   }
 
   &__related-content {
-    border-top: 1px solid lighten($color-primary, 30%);
+    border-top: 2px solid lighten($color-primary, 40%);
     margin: $base-line-height 0 0;
     padding: $base-line-height 0 0;
+
+    .o-media__headline {
+      font-weight: 500;
+      font-size: 1.14rem;
+    }
+
+    > .heading {
+      text-transform: uppercase;
+      margin-bottom: 1.375rem;
+    }
+
+    .c-list--bullet {
+      padding: 0;
```

Comment:
I know the design calls for the bullets on related links to flow out into the left gutter outside the media object. I'm not sure I 100% agree with this choice, but if you're going to implement it you need a media query to add the padding back in when you get to smaller screens - otherwise you risk having the bullets flow off-canvas.